backend/api/Mapping/TypeMap.cs

```diff
@@ -20,6 +20,12 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.IsDisabled, src => src.IsDisabled)
                 .Map(dest => dest.DisplayOrder, src => src.DisplayOrder);
 
+            config.NewConfig<Model.TypeModel<short>, Entity.ICodeEntity<short>>()
+                .Map(dest => dest.Code, src => src.Id)
+                .Map(dest => dest.Description, src => src.Description)
+                .Map(dest => dest.IsDisabled, src => src.IsDisabled)
+                .Map(dest => dest.DisplayOrder, src => src.DisplayOrder);
+
```

Comment:
Missing mapping

Reply:
the catch with this is that we need to make sure that when updating the type model should not be created most of the time

Reply:
I will remove it then to avoid bugs - we can re-add if needed